mta-sts.md

```diff
@@ -148,24 +148,22 @@ An example TXT record is as below:
 
 `_mta-sts.example.com.  IN TXT "v=STSv1; id=20160831085700Z;"`
 
-The formal definition of the `_mta-sts` TXT record, defined using [@!RFC5234],
+The formal definition of the `_mta-sts` TXT record, defined using [@!RFC7405],
 is as follows:
 
-    sts-text-record = sts-version *WSP field-delim *WSP sts-id
+    sts-text-record = sts-version field-delim sts-id
                       [field-delim [sts-extensions]]
```

Comment:
We can simplify this my replacing [field-delim [sts-extensions]] with 
*(field-delim sts-extension)

and remove line 166:
sts-extensions  = sts-extension *(field-delim sts-extension)
                        [field-delim]                      ; extension fields